distribution/bin/generate-binary-license.py

```diff
@@ -102,7 +102,7 @@ def print_license(license):
                     license_phrase += "see {}".format(each_file)
                 else:
                     license_phrase += ", {}".format(each_file)
-    
```

Comment:
nit these changes looks accidental.

Reply:
@cryptoe That's just my editor auto removing trailing whitespace.

Reply:
We generally try to avoid such changes.